Readme.md

```diff
@@ -84,6 +84,7 @@ attacker$ssh -p 31337 <LHOST>
 ```shell
 # On attacker (get ready to catch the incoming request;
 # can be omitted if you already have an ssh daemon running, e.g. OpenSSH)
+# NOTE: LPORT must NOT be 8888, as it'll cause a conflict.
```

Comment:
Hi @PinkDev1 ,

sorry for my late response, I was abroad the last few weeks.

Thanks for your PR, I agree that your note helps others!
I don't quite see the "must not" part though: by using the the `-b` flag on the victim it still works (but admittedly causes problems with the ssh config).

Are you OK with my suggestion below?

```suggestion
# NOTE: LPORT of 8888 collides with incoming connections; use the flag `-b 8889` or similar on the victim in that case
```